### ISSUE TITLE
Two files (which shouldn't?) are changed during integration tests

### ISSUE THREAD

**Issue by breznak** [numenta/nupic-legacy#856]
> examples/opf/experiments/multistep/datasets/first_order_0.csv modified: 

I'm surprised what looks like data input file is being touched? (many changes in values)

> tests/integration/py2/nupic/engine/data/vectorfile.nta/network.yaml

Just a whitespace change, still appears in git diff. 

Edit: to trigger, run `make tests_run_all`

**Comment by scottpurdy** [comment]
@mihail911 - Did you have a fix for this?

**Comment by vitaly-krugl** [comment]
![screen shot 2015-08-11 at 2 32 58 pm](https://cloud.githubusercontent.com/assets/1441066/9211451/2c23744c-4037-11e5-961c-8f46b617a10a.png)

Looking at the screen capture of of searching the repo for "first_order_0.csv",  it looks like `first_order_0.csv` is generated by the make_datasets.py script, which is invoked by opf_experiment_results_test.py here: https://github.com/numenta/nupic/blob/fea73884669423d6f73a2b5b638d3410f97653c1/tests/integration/nupic/opf/opf_experiment_results_test.py#L84.

**Comment by rhyolight** [comment]
This has always annoyed me!

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by mihail911** [comment]
Yes, I have noticed this as well. I was working on a fix last week before I got sidetracked with other stuff. The lines that @vitaly-krugl referenced make a system call to make_datasets.py which regenerates the first_order_0.csv file every time those tests are run. Given that the values generated in the file are not done deterministically, it makes sense that there would be runs where a different first_order_0.csv is created. I don't think it is necessary to constantly regenerate a data file, especially if this data file is being shipped with the nupic repo, as is the case with first_order_0.csv. The temporary fix I was working on was  simply removing that data file-generating system call in opf_experiment_results and then migrating first_order_0.csv to a more permanent location outside of nupic/examples/opf/experiments/multistep/datasets, ideally somewhere within nupic/datafiles. Not sure if someone has a better fix...
